src/components/Scheme/SchemeProposalsPage.tsx

```diff
@@ -134,20 +135,36 @@ class SchemeProposalsPage extends React.Component<IProps, null> {
             <div className={css.proposalsHeader}>
               No upcoming proposals
             </div>
-            <p>You can be the first one to create a {schemeFriendlyName} proposal today! :)</p>
-            <div className={css.cta}>
-              <Link to={"/dao/" + daoState.address}>
-                <img className={css.relax} src="/assets/images/lt.svg"/> Back to schemes
-              </Link>
-              <a className={classNames({
-                [css.blueButton]: true,
-                [css.disabled]: !isActive,
-              })}
-              href="#!"
-              onClick={isActive ? this._handleNewProposal : null}
-              data-test-id="createProposal"
-              >+ New Proposal</a>
-            </div>
+            { isWikiScheme ? 
+              <>
+                <p>You can be the first one to create a Wiki merge proposal today! (:</p>
+                <div className={css.cta}>
+                  <Link to={"/dao/" + daoState.address}>
+                    <img className={css.relax} src="/assets/images/lt.svg"/> Back to schemes
+                  </Link>
+                  <Link to={"/dao/" + daoState.address + "/wiki"}>
```

Comment:
Should you be checking `isActive` here, like the code below does?

Reply:
This guarantees that the scheme is 100% functional, right? If the answer is yes, then yup, we should definitely add this validation

Reply:
It checks whether the voting machine is active.

Reply:
Now that I think a bit more about this - I don't think we should add this validation, because users can not create a proposal on this scheme through alchemy's UI; it can only be triggered from the wiki components. And doing this here, we would just remove the button to go to the wiki from the scheme page, not from the sidebar. 

My conclusion is that we shouldn't add this. Thoughts?

Reply:
If the voting machine is not activated then any attempt to create a proposal will fail.  So I think there are two options:  1) Do the check here *and* in the sidebar, or 2) do the check inside the wiki code.

Reply:
Okok, adding this now

Reply:
I don't see what you did for this.